doc/man3/EVP_sm3.pod

```diff
@@ -0,0 +1,57 @@
+=pod
+
+=head1 NAME
+
+EVP_sm3
+- SM3 for EVP
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ const EVP_MD *EVP_digestname(void)
```

Comment:
doc-nits complains about this missing in the NAME section.  In this case, it's just one name, so I think it would be better to simply have `EVP_sm3` instead of `EVP_digestname` here...

Reply:
Addressed.